README.md

```diff
@@ -1,6 +1,6 @@
 # Apollo Voyager Server
```

Comment:
We will still using `Apollo` here? I see the packages still using it, but the repo don't

Reply:
There is this ticket. Initially assigned to @darahayes.
I picked that up now.

https://issues.jboss.org/browse/AEROGEAR-8432